crates/libtiny_tui/src/messaging.rs

```diff
@@ -39,6 +39,9 @@ pub(crate) struct MessagingUI {
 
     /// Last timestamp added to the UI.
     last_ts: Option<Timestamp>,
+
+    // Autocompletion character to be used when at the start of the line. Set with `set_completion_char`.
+    completion_char: Option<String>,
```

Comment:
I think it would make more sense to move this field into `InputArea`. Then you can avoid passing the `completion_char` into the `autocomplete()` function above, and just use `self.completion_char`.

Reply:
Done